client/app/src/accounts/account.controller.js

```diff
@@ -1275,14 +1276,31 @@
           .cancel(gettextCatalog.getString('Cancel'))
         $mdDialog.show(confirm).then(() => {
           networkService.removeNetwork(network)
-          self.listNetworks = networkService.getNetworks()
-          toastService.success(gettext('Network removed succesfully!'), 3000)
+          if (isActive) {
+            dialogService.openLoadingDialog(self.currentTheme,
+                                            gettext('Network removed succesfully'),
+                                            gettext('Network removed succesfully - will now switch the network'))
+            setTimeout(() => networkService.switchNetwork(null, true), 4000)
```

Comment:
Maybe halve the timeout as felt a bit long for me, it was enough time for me to wonder if it was going to reload.

Reply:
oh maybe take a look at my other comment. instead of doing a timeout, https://github.com/ArkEcosystem/ark-desktop/pull/566#discussion_r169040742